### PR TITLE
[2.14.x] DDF-4709 fixed session memory cleanup

### PR DESCRIPTION
**Pull request by stustison**
#### Ask 2 committers to review/merge the PR and tag them here.

@bdeining
@blen-desta
@garrettfreibott
@pklinef
@rzwiefel
@stustison
@troymohl

#### What are the relevant tickets?
For Jira:
[](https://codice.atlassian.net/browse/)

For GH Issues:
Fixes: #____

#### Checklist:
- [ ] Documentation Updated
- [ ] Update / Add Threat Dragon models
- [ ] Update / Add Unit Tests
- [ ] Update / Add Integration Tests

#### Notes on Review Process
Please see [Notes on Review Process](https://codice.atlassian.net/wiki/spaces/DDF/pages/71946981/Pull+Request+Guidelines) for further guidance on requirements for merging and abbreviated reviews. 

#### Review Comment Legend:
- ✏️ (Pencil) This comment is a nitpick or style suggestion, no action required for approval. This comment should provide a suggestion either as an in line code snippet or a gist. 
- ❓ (Question Mark) This comment is to gain a clearer understanding of design or code choices, clarification is required but action may not be necessary for approval.
- ❗ (Exclamation Mark) This comment is critical and requires clarification or action before approval.

### CODE REVIEW

**Comment by stustison** [comment]
build now

File: platform/platform-paxweb-jettyconfig/src/main/java/org/codice/ddf/security/session/AttributeSharingHashSessionIdManager.java [hunk 1]
```diff
@@ -175,25 +188,21 @@ protected void provideNewSessionAttributes(
       String id,
       Map<String, Object> sessionAttributes) {
     // Make sure these attributes are different than the latest.
-    if (sessionAttributes != null && !sessionAttributes.equals(getLatestSessionAttributes(id))) {
```
Comment (lamhuy) [inline platform/platform-paxweb-jettyconfig/src/main/java/org/codice/ddf/security/session/AttributeSharingHashSessionIdManager.java]: should this comment be updated as well as java doc above?

**Comment by stustison** [comment]
build now

**Comment by jrnorth** [comment]
build now

**Comment by stustison** [comment]
build now

**Comment by rzwiefel** [comment]
@stustison Any update on this PR?

**Comment by bdeining** [comment]
build now

**Comment by shaundmorris** [comment]
Merging the PR since the 2.14.x branch no longer has PR builds.

**Pull request merged**